_stbt/imgproc_cache.py

```diff
@@ -0,0 +1,211 @@
+import functools
+import inspect
+import json
+import os
+import sys
+from contextlib import contextmanager
+
+import numpy
+
+from _stbt.gst_utils import Gst, numpy_from_sample
+from _stbt.logging import ImageLogger
+from _stbt.utils import mkdir_p
+
+_cache = None
+
+
+@contextmanager
+def cache(filename=None):
+    if os.environ.get('STBT_DISABLE_CACHING'):
+        yield
+        return
+
+    global _cache
+
+    sys.path.append(os.path.dirname(__file__))
```

Comment:
Is this `sys.path.append` necessary? We have `from _stbt.gst_utils import ...` above, so if that works why wouldn't this work?